book/website/_toc.yml

```diff
@@ -96,6 +96,8 @@ chapters:
       file: reproducible-research/rdm/rdm-dmp
     - title: The FAIR Principles and Practices
       file: reproducible-research/rdm/rdm-fair
+    - title: Personal data management
+      file: reproducible-research/rdm/rdm-personal
```

Comment:
The file name for the section needs to be changed from `personal.md` to `rdm-personal.md` I think. It builds on Netlify, but doesn't appear linked to any section: https://deploy-preview-1471--the-turing-way.netlify.app/reproducible-research/rdm/personal.html

Reply:
Good catch. @EstherPlomp you can rename the file directly by editing the file name here:
<img width="894" alt="image" src="https://user-images.githubusercontent.com/5370471/128005038-60e8e0c2-fde2-4c42-a024-84461ac2d83d.png">

Reply:
Done, thanks for spotting!